src/runtime_src/tools/scripts/pkgapu.sh

```diff
@@ -105,7 +105,7 @@ EOF
 	cp $dir/RPMS/noarch/*.rpm $PACKAGE_DIR
 }
 
-SYSTEM_DTB_ADDR="0x1000"
+SYSTEM_DTB_ADDR="0x40000"
```

Comment:
Hope RPU package is copying the system.dtb is at 0x40000 address.

Reply:
Yes. David test it. RPU is able to copy system.dtb to 0x40000.